runtime/derive/src/lib.rs

```diff
@@ -72,21 +72,21 @@ fn asc_type_derive_struct(item_struct: ItemStruct) -> TokenStream {
                 #(bytes.extend_from_slice(&self.#field_names.to_asc_bytes()?);)*
 
                 // Assert that the struct has no padding.
-                assert_eq!(bytes.len(), size_of::<Self>());
+                assert_eq!(bytes.len(), std::mem::size_of::<Self>());
```

Comment:
Nice! I actually wanted to change our derive crate to use more full names, it's a good practice in proc macros.

Reference: https://github.com/dtolnay/proc-macro-workshop/blob/master/builder/tests/09-redefined-prelude-types.rs

I mean, only our code uses it, but I guess it's probably better to avoid conflicting names errors.

Reply:
Yes definitely bad practice to use names that are not fully qualified.